app/uk/gov/hmrc/merchandiseinbaggage/views/CheckYourAnswersImportView.scala.html

```diff
@@ -198,13 +198,10 @@ <h2 class="govuk-heading-m">@messages(s"checkYourAnswers.${agentOrTrader}.sendDe
 
   <p class="govuk-body">@messages("checkYourAnswers.sendDeclaration.confirm")</p>
 
- <div class="govuk-warning-text">
-  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-  <strong class="govuk-warning-text__text">
-   <span class="govuk-warning-text__assistive">@messages("checkYourAnswers.sendDeclaration.warning")</span>
-   @messages(s"checkYourAnswers.sendDeclaration.warning.Import.${agentOrTrader}.message")
-  </strong>
- </div>
+  @govukWarningText(WarningText(
+     iconFallbackText = Some(messages("checkYourAnswers.sendDeclaration.warning")),
+     content = Text(messages(s"checkYourAnswers.sendDeclaration.warning.Import.${agentOrTrader}.message"))
+  ))
```

Comment:
Issues with warning text after upgrade as it was showing the `Warning` text on the page. Used GovukWarningText from library which fixes the issue.